tests/TestCase/Controller/SentencesControllerTest.php

```diff
@@ -295,12 +295,44 @@ public function testPaginateRedirectsPageOutOfBoundsToLastPage() {
                 'user_id' => 1,
             ];
         }
-        $sentences = TableRegistry::get('Sentences');
+        $sentences = TableRegistry::getTableLocator()->get('Sentences');
         $entities = $sentences->newEntities($newSentences);
-        $result = $sentences->saveMany($entities);
+        $sentences->saveMany($entities);
+
+        $nbSentences = count($sentences->find()->where(['user_id' => $userId])->all());
+        $lastPage = ceil($nbSentences / $defaultNbPerPage);
 
         $this->get("/eng/sentences/of_user/$user?page=9999999");
+        $this->assertRedirect("/eng/sentences/of_user/$user?page=$lastPage");
+    }
+
+    public function testPaginateRedirectsPageOutOfBoundsToLastPage_withUserSetting() {
+        $user = 'kazuki';
+        $userId = 7;
+        $users = TableRegistry::getTableLocator()->get('Users');
+        $NbPerPageSetting = $users->getSettings($userId)['settings']['sentences_per_page'];
```

Comment:
Could you rename this variable so that it starts with a lowercase? Let’s keep the same naming conventions within a single file.